forum/templates/users/password_reset.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Just recommendation(If not necessary then just skip this comment)

Add meta tag to ensure the page is responsive on mobile devices
<meta name="viewport" content="width=device-width, initial-scale=1.0">